tests/torch/nncf_network/helpers.py

```diff
@@ -0,0 +1,186 @@
+from nncf.torch.layer_utils import COMPRESSION_MODULES
+from tests.torch.helpers import DummyOpWithState
+
+
+class SimplestModel(torch.nn.Module):
+    INPUT_SIZE = [1, 1, 32, 32]
+
+    def __init__(self):
+        super().__init__()
+        self.conv = torch.nn.Conv2d(1, 1, 1)
+
+    def forward(self, x):
+        return self.conv(x)
+
+
+AVAILABLE_TARGET_TYPES = (
+    TargetType.OPERATION_WITH_WEIGHTS,
+    TargetType.OPERATOR_PRE_HOOK,
+    TargetType.OPERATOR_POST_HOOK,
+    TargetType.PRE_LAYER_OPERATION,
+    TargetType.POST_LAYER_OPERATION,
+)
+
+
+class InsertionCommandBuilder:
+    """
+    Contains methods which allows to build all possible commands
+    for the given torch.nn.Module. Target module should have
+    NNCF_CONV_NODES_NAMES and CONV_NODES_NAMES with names of
+    target model convolutions and names of nncf-wrapped target model convolutions
+    """
+
+    def __init__(self, model_cls: Type[torch.nn.Module]):
+        self.model_cls = model_cls
+
+    TRACE_VS_NODE_NAMES = {True: "CONV_NODES_NAMES", False: "NNCF_CONV_NODES_NAMES"}
+
+    def create_pt_insertion_command(
+        self,
+        target_type: TargetType,
+        priority: TransformationPriority,
+        trace_parameters: bool,
+        fn: Optional[torch.nn.Module] = None,
+        group: str = "default_group",
+        op_unique_name: Optional[str] = None,
+    ):
+        attr_name = self.TRACE_VS_NODE_NAMES[trace_parameters]
+        target_point = PTTargetPoint(
+            target_type=target_type, target_node_name=getattr(self.model_cls, attr_name)[0], input_port_id=0
+        )
+        if fn is None:
+            fn = DummyOpWithState("DUMMY_STATE")
+        return PTInsertionCommand(point=target_point, fn=fn, priority=priority, hooks_group_name=group)
+
+    def create_pt_shared_fn_insertion_command(
+        self,
+        target_type: TargetType,
+        priority: TransformationPriority,
+        trace_parameters: bool,
+        compression_module_type: ExtraCompressionModuleType,
+        fn: Optional[torch.nn.Module] = None,
+        group: str = "default_group",
+        op_unique_name: str = "UNIQUE_NAME",
+    ):
+        target_points = []
+        attr_name = self.TRACE_VS_NODE_NAMES[trace_parameters]
+        for node_name in getattr(self.model_cls, attr_name):
+            target_points.append(PTTargetPoint(target_type=target_type, target_node_name=node_name, input_port_id=0))
+        if fn is None:
+            fn = DummyOpWithState("DUMMY_STATE")
+        return PTSharedFnInsertionCommand(
+            target_points=target_points,
+            fn=fn,
+            compression_module_type=compression_module_type,
+            op_unique_name=op_unique_name,
+            priority=priority,
+            hooks_group_name=group,
+        )
+
+    def get_command_builders(self):
+        """
+        Get all command builders available in a tuple.
+        """
+        return (
+            self.create_pt_insertion_command,
```

Comment:
return tuple (FN, Type of command) 
remove COMMAND_CLASSES

Reply:
Done